### ISSUE TITLE
This library isn't using V2 of the Harvest API

### ISSUE THREAD

**Issue by BillBrower** [GildedHonour/harvest-api-client#1]
If I made a pull-request that updated this library so that it used V2 of the Harvest API would you accept it?

**Comment by GildedHonour** [comment]
@BillBrower yes

**Comment by BillBrower** [comment]
Awesome, I'll work on that today! In `harvest-api-client/harvest.py` I see that you import `requests` but I don't see where it is used. Would you be opposed to me refactoring the `_request` method on the `Harvest` class so that it just uses `requests` instead of `urllib` then we could axe both `urllib` and `json` as dependencies.

**Comment by GildedHonour** [comment]
@BillBrower go ahead.

**Comment by rmeyer-taos** [comment]
Any progress on this issue?

**Comment by BillBrower** [comment]
@rmeyer-taos No, I never ended up having time to make a PR. I did end up modifying this client to pull information from V2 of the Harvest API though. Here is the [code](https://github.com/builtbykrit/albatross-api/blob/master/albatross/harvest/utils.py) if you are interested. It isn't fully featured, or anywhere close to being PR ready, but if you just need to pull information from Harvest then it should be a start.

**Comment by rmeyer-taos** [comment]
@BillBrower thanks for the response and the info. Depending on what horrible things I am told to do with this integration I might take you up on using some of your code.
